api-inference-community/docker_images/generic/app/pipelines/text_to_image.py

```diff
@@ -0,0 +1,16 @@
+from app.pipelines import Pipeline
+
+
+class TextToImagePipeline(Pipeline):
```

Comment:
If you add only dummy stuff like this everywhere in `generic` why not just use `Pipeline` everywhere instead of having dummy subclass that does nothign ?

Reply:
That was the initial plan, but unfortunately it can be the case that different pipelines require somewhat different behaviour. For example, the audio pipelines also required specifying sampling rate (this is done in #266). 

Maybe we could have a `DefaultPipeline`, `AudioPipeline`, etc. WDYT?